addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2489,6 +2489,12 @@ options.registry.DeviceVisibility = options.Class.extend({
      * @see this.selectClass for parameters
      */
     async toggleDeviceVisibility(previewMode, widgetValue, params) {
```

Comment:
If my eyes do not deceive me, there is more than 72 characters per line in your commit message :)